lib/eco/treemanager.py

```diff
@@ -127,6 +128,8 @@ def right(self):
 
     def jump_left(self):
         self.node = self.find_previous_visible(self.node)
+        if self.node.symbol.name == "\r":
+            self.line -= 1
```

Comment:
It won't because the code I commited to master is a bit different, as this code still had some bugs. So @snim2 will have to resolve the conflicts by hand.